nuxeo-runtime/nuxeo-runtime-metrics/src/main/java/org/nuxeo/runtime/metrics/MetricsDescriptor.java

```diff
@@ -81,6 +86,23 @@ public MetricsDescriptor() {
 
         public static final String PREFIX_PROPERTY = "metrics.graphite.prefix";
 
+        /**
+         * A list of metrics that if defined should be kept reported, separated by commas
+         * 
+         * @since 9.3
+         */
+        public static final String FILTER_PROPERTY = "metrics.graphite.filter";
+
+        /**
+         * @since 9.3
+         */
+        public static final String DEFAULT_METRICS = "nuxeo.cache.user-entry-cache,nuxeo.cache.group-entry-cache,nuxeo.directories.userDirectory,nuxeo.directories.groupDirectory";
+
```

Comment:
this is a good default for caches and directories, but there are much more metrics to add